src/pages/resultsView/pathwayMapper/PathWayMapperContainer.tsx

```diff
@@ -27,8 +40,51 @@ const PathWayMapperContainer: React.FunctionComponent<IPathwayMapperContainerPro
             activeTab:
                 urlWrapper.query.pathways_source ||
                 ResultsViewPathwaysSubTab.PATHWAY_MAPPER,
+            tooltipContent: null as DatabaseItem[] | null,
+            isFetchingTooltip: false,
         }));
 
+        useEffect(() => {
```

Comment:
it would be more consistent with codebase to use a mobxpromise here to manage the loading behavior.  Look elsewhere in codebase for remoteData( ... ).    This can be assigned to the store above.   e.g.

```
{
       someData:makeRemoteData()
}

```
where makeRemoteData is a function local to this file which returns remoteData(...)
```